src/main/webapp/dev/js/common/bootboxWrapper.es6

```diff
@@ -28,12 +28,16 @@ function showModalAlert(titleText, messageText, okButtonText, color) {
         buttons: {
             okay: {
                 label: okButtonText || DEFAULT_OK_TEXT,
-                className: `modal-btn-ok btn-${color}` || StatusType.DEFAULT,
+                className: `modal-btn-ok btn-${color || StatusType.DEFAULT}`,
             },
         },
     })
-    // applies bootstrap color to title background
-    .find('.modal-header').addClass(`alert-${color}` || StatusType.DEFAULT);
+    .find('.modal-header')
```

Comment:
This would avoid unnecessary repetition of code. Also, the comments can then be dropped as the code would become self-documenting :)